lib/crowbar/client/command/node/list.rb

```diff
@@ -73,6 +73,10 @@ def content_from(request)
                 )
```

Comment:
Not sure I understand what you are saying, because I can find a lot of your code which doesn't split the parameters onto a separate line.  How do you decide when to split and when not to split?